### ISSUE TITLE
Torrent-stream/jPlayer esque API

### ISSUE THREAD

**Issue by iadj** [mafintosh/playback#56]
Something I really like about Torrent-Stream, Webtorrent, and jPlayer is its simple to use API. The ability to easily programmatically add/play content and download/seed torrents would allow for easy client customization.

i.e:

`player.addToPlayList({
URL: 'http://xxx.xxx/aaa.torrent', 
autoPlay: true, 
cacheLocation: 'file://${__dirname}/torrentCache'
})`

and

`player.play('URL')`

**Comment by freeall** [comment]
So you mean it should run as a daemon and you could connect to it and add to the playlist, etc?

**Comment by iadj** [comment]
@freeall Not necessarily although that's a great idea! My thoughts were to have an API within the player app itslef. I feel like lots of opportunities for customization open up when there are some torrent-stream like functions available for Playback.

**Comment by freeall** [comment]
I think I understand what you want, but you need to connect to the app somehow? Or did you want to create a wrapper around the application that does something and then adds to the playlist?

**Comment by iadj** [comment]
@freeall Yeah I'd like to try wrapping some things around the Playback player
